packages/expo-screen-capture/src/ExpoScreenCapture.web.ts

```diff
@@ -0,0 +1,14 @@
+export default {
+  get name(): string {
+    return 'ExpoScreenCapture';
+  },
+  get activatePreventScreenCapture(): null {
```

Comment:
match the android and iOS interfaces:
```suggestion
  get async activatePreventScreenCapture(): Promise<null> {
```

Reply:
done, should we suffix the method names with `Async`?

Reply:
I'd say so, since we want to communicate that this is an async function and could throw an error asynchronously. If you don't await the function call, you'll get an unhandled promise rejection; you need to await in order to catch.